src/index.ts

```diff
@@ -155,8 +160,10 @@ class MultinetAPI {
     }
 
     return this.client.post(`/${type}/${workspace}/${table}`, text, {
-      headers: { 'Content-Type': 'text/plain' },
+      ...config,
```

Comment:
What happens if `config` is undefined (same question for `params` a couple of lines below)?

Reply:
Using the spread operator on `undefined` simply inserts nothing into the object.

Reply:
Huh, TIL.